.coveragerc

```diff
@@ -137,7 +137,10 @@ omit =
     homeassistant/components/broadlink/updater.py
     homeassistant/components/brottsplatskartan/sensor.py
     homeassistant/components/browser/*
+    homeassistant/components/brunt/__init__.py
     homeassistant/components/brunt/cover.py
+    homeassistant/components/brunt/const.py
+    homeassistant/components/bsblan/__init__.py
```

Comment:
This line doesn't seem related to this PR.

Reply:
I think this is my mistake during a merge, I will check